portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,11 +26,23 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
  
 /** Servlet that adds comments to datastore */
 @WebServlet("/data") 
 public class DataServlet extends HttpServlet 
 { 
+  private LanguageServiceClient languageService;
+
+  public void init() throws ServletException {
+    try {
+        languageService = LanguageServiceClient.create();
+    } catch (Exception e) {
```

Comment:
Can we make sure that we don't generically catch all `Exception`s so that we aren't bucketing everything as an IO error?

Reply:
Oh I'm sorry, I recall yesterday that I sent a commit from my earlier branch to this branch by accident, so the DataServlet file is behind the edits that you suggested. In the previous branch that was reviewed I wrote a more generic error message.

Reply:
I see that I am sending commits to the wrong branch when I have multiple branches open for review, I'll work on keeping only one branch open, that way I do not send old files accidentally

Reply:
Keeping this open since it still looks like we're catching all Exceptions atm.